geomesa-spark-jts/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>geomesa_2.11</artifactId>
```

Comment:
would it be a problem to move this under `geomesa-spark` instead of a top-level module?

Reply:
Wouldn't be a problem. I think the rational of having it as a top-level module was a step towards moving it out of geomesa entirely. @jnh5y might have more thoughts.

Reply:
that probably won't happen soon :) It's easy to move either way I'd claim.